examples/textures/textures_image_loading.zig

```diff
@@ -34,7 +35,6 @@ pub fn main() anyerror!void {
     // De-Initialization
     //--------------------------------------------------------------------------------------
     defer rl.unloadTexture(texture); // Texture unloading
```

Comment:
Let's put the `closeWindow()` call right above the `unloadTexture` so it's still within the De-Initialization section